src/resources/views/fields/toggle.blade.php

```diff
@@ -0,0 +1,103 @@
+<!-- radio toggle -->
+@php
+    $optionPointer = 0;
+    $optionValue = old($field['name']) ? old($field['name']) : (isset($field['value']) ? $field['value'] : (isset($field['default']) ? $field['default'] : '' ));
+@endphp
+
+<div @include('crud::inc.field_wrapper_attributes') >
+
+    <div>
+        <label>{!! $field['label'] !!}</label>
+    </div>
+
+    @if( isset($field['options']) && is_array($field['options']) )
+
+        @foreach ($field['options'] as $value => $label )
+            @php ($optionPointer++)
+
+            @if( isset($field['inline']) && $field['inline'] )
+
+            <label class="radio-inline" for="{{$field['name']}}_{{$optionPointer}}">
+                <input data-field-toggle="{{ json_encode($field['hide_when']) }}" type="radio" id="{{$field['name']}}_{{$optionPointer}}" name="{{$field['name']}}" value="{{$value}}" {{$optionValue!=='' && $optionValue == $value ? ' checked': ''}}> {!! $label !!}
```

Comment:
`$optionValue !== ''` - notice the spaces (may as well be consistent with the following ==).